tests/e2e-openshift/Dockerfile

```diff
@@ -26,7 +26,7 @@ RUN curl -LO https://github.com/kudobuilder/kuttl/releases/download/v0.15.0/kube
     && mv kubectl-kuttl_0.15.0_linux_x86_64 /usr/local/bin/kuttl
 
 # Install Chainsaw e2e
-RUN go install github.com/kyverno/chainsaw@v0.1.6
+RUN go install github.com/kyverno/chainsaw@v0.1.7
```

Comment:
Might be easier to run `make chainsaw` here and avoid updating the version in two places.